### ISSUE TITLE
NC13.0.0: Share view when error; double logo + wrong aspect ratio

### ISSUE THREAD

**Issue by jallakim** [nextcloud/server#8458]
### Steps to reproduce
1. Have NC13 with custom logo;

![avatar](https://user-images.githubusercontent.com/359918/36439904-8692e786-166e-11e8-9778-339c3d6ddce6.png)

2. Visit share link that contains an error (for example wrong share token).
3. Error message is shown, containing "double" logo + wrong aspect ratio

![dump 2018-02-20 at 18 41 02](https://user-images.githubusercontent.com/359918/36439762-20471484-166e-11e8-83ad-bdf43b0a7843.png)

### Expected behaviour
Logo should be displayed correctly (once) and with correct aspect ratio

### Actual behaviour
Logo is displayed twice, and one of them seems to be "skewed/stretched" (i.e. wrong aspect ratio).

### Server configuration

**Operating system**: CentOS 7

**Web server:** Apache2.4.6

**Database:** Postgresql 9.2.23

**PHP version:** 7.0.27

**Nextcloud version:** 13.0.0 (final/stable)

**Updated from an older Nextcloud/ownCloud or fresh install:** Fresh install.

**Where did you install Nextcloud from:** .tar.gz-file downloaded from download.nextcloud.com

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - activity: 2.6.1
  - comments: 1.3.0
  - dav: 1.4.6
  - federatedfilesharing: 1.3.1
  - federation: 1.3.0
  - files: 1.8.0
  - files_pdfviewer: 1.2.0
  - files_sharing: 1.5.0
  - files_texteditor: 2.5.1
  - files_trashbin: 1.3.0
  - files_versions: 1.6.0
  - files_videoplayer: 1.2.0
  - firstrunwizard: 2.2.1
  - gallery: 18.0.0
  - groupfolders: 1.2.0
  - logreader: 2.0.0
  - lookup_server_connector: 1.1.0
  - nextcloud_announcements: 1.2.0
  - notifications: 2.1.2
  - oauth2: 1.1.0
  - password_policy: 1.3.0
  - provisioning_api: 1.3.0
  - serverinfo: 1.3.0
  - sharebymail: 1.3.0
  - sharerenamer: 1.3
  - survey_client: 1.1.0
  - systemtags: 1.3.0
  - theming: 1.4.1
  - twofactor_backupcodes: 1.2.3
  - updatenotification: 1.3.0
  - workflowengine: 1.3.0
Disabled:
  - admin_audit
  - encryption
  - files_external
  - user_external
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "domain.foo.bar",
            "otherdomain.foo.bar",
        ],
        "apps_paths": [
            {
                "path": "\/srv\/vhosts\/nextcloud\/web\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/srv\/vhosts\/nextcloud\/web\/apps2",
                "url": "\/apps2",
                "writable": true
            }
        ],
        "versions_retention_obligation": "disabled",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/domain.foo.bar",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "dbtype": "pgsql",
        "version": "13.0.0.14",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "proxy": "proxy.foo.bar:3128",
        "maintenance": false,
        "loglevel": 1
    }
}
```
</details>

**Are you using external storage, if yes which one:** Local storage.

**Are you using encryption:** No.

**Are you using an external user-backend, if yes which one:** No external user-backend.

### Client configuration
**Browser:** Chrome 63.0.3239.132 (Official Build) (64-bit)

**Operating system:** OS X 10.12.6 (16G29)

#### Web server error log
<details>
<summary>Web server error log</summary>

```
No error logs.
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
No error logs.
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
No error logs.
```
</details>

**Comment by jallakim** [comment]
This is exactly the same as in #7575, but that issue (and it's fixes) only includes the share link with password. The issue is still present in the latest/final release for links that generates an error message.

**Comment by MorrisJobke** [comment]
> This is exactly the same as in #7575, but that issue (and it's fixes) only includes the share link with password. The issue is still present in the latest/final release for links that generates an error message.

It uses the same CSS and thus #7678 should fix the issue as well. The fix is not yet in a released version and will be shipped in 13.0.1.

As a workaround you could apply this change to your instance: https://github.com/nextcloud/server/pull/7678/files

**Comment by jallakim** [comment]
@MorrisJobke, this is not true. The 13.0.0 release contains the fix you're referring to;

```
jocke@noshut:/tmp$ wget -q 'https://download.nextcloud.com/server/releases/nextcloud-13.0.0.zip'
jocke@noshut:/tmp$ unzip -q nextcloud-13.0.0.zip 
jocke@noshut:/tmp$ grep -n "border-box" nextcloud/apps/files_sharing/css/authenticate.css
11:	box-sizing: border-box;
jocke@noshut:/tmp$ grep -n "'core', 'guest'" nextcloud/apps/files_sharing/templates/authenticate.php
4:	style('core', 'guest');
```

It still has the defect on the error page, so there's obviously something more that needs fixing. Please reopen this issue.

**Comment by MorrisJobke** [comment]
> @MorrisJobke, this is not true. The 13.0.0 release contains the fix you're referring to;

Ah right - then I mixed this up.

@juliushaertl @skjnldsv Could you have a look at this?

**Comment by skjnldsv** [comment]
Have you cleared your cache? :)

**Comment by jallakim** [comment]
This was a fresh install on a "new" FQDN, so my computer wouldn't have anything in it's cache. I just did a "Empty Cache and Hard Reload" in my Chrome, and it's still an issue.

**Comment by skjnldsv** [comment]
Okay, thanks! :)
We had some issues with that before, I needed to be sure.
Could you right click the image and click inspect element. Select the `<div id="header">`, right click, copy element and paste it here please :)

**Comment by jallakim** [comment]
```
<div id="header">
							<div class="logo">
								<h1 class="hidden-visually">
									foobar								</h1>
																	<img src="/index.php/apps/theming/logo?v=16">
															</div>
						</div>
```

**Comment by skjnldsv** [comment]
No problem! It's perfectly normal :) 
There should not be an img element in here, that's why it's bugged. @juliushaertl how is that possible? Didn't we removed this?

**Comment by juliushaertl** [comment]
The image is there to provide proper scaling and was added in https://github.com/nextcloud/server/pull/4632 

The issue should be fixed with https://github.com/nextcloud/server/pull/8286 (which is merged for 13.0.1)
